src/Routes.tsx

```diff
@@ -1,15 +1,18 @@
-import React from 'react';
 import { Switch, Route } from 'react-router-dom';
 
 import Landing from './pages/Landing';
 import Login from './pages/Login';
 import Register from './pages/Register';
+import EditProfile from './pages/EditProfile';
+import EmailVerification from './pages/EmailVerification';
 
 const Routes = () => (
 	<Switch>
 		<Route exact path='/' component={Landing} />
 		<Route path='/login' component={Login} />
 		<Route path='/register' component={Register} />
+		<Route path='/verify' component={EmailVerification} />
+		<Route path='/edit-profile' component={EditProfile} />
```

Comment:
Specify if this is editJobSeekerProfie or editCompanyProfile. They have different pages.

Reply:
I will include it as specified. Anything else prior to merging?